Consumer/MessageConsumer.php

```diff
@@ -74,9 +75,14 @@ public function execute(AMQPMessage $msg)
             throw new \InvalidArgumentException('You have to specify Domain class name');
         }
         /** @var MessageInterface $message */
-        $message = $this->serializer
-            ->deserialize($msg->body, $this->messageClassName, 'json');
-        $this->messageDispatcher->dispatchMessage($message);
+        try {
+            $message = $this->serializer
+                ->deserialize($msg->body, $this->messageClassName, 'json');
+            $this->messageDispatcher->dispatchMessage($message);
+        } catch (LifecircleException $e) {
+            return $e->getFlag();
+        }
+        
         return true;
```

Comment:
Is it mistake in parent interface description or in this implementation?
`@return` description inside PHPDoc is very confusing. Even though it is copied from ConsumerInterface, and might be wrong there, it should be update with accordance in our implementation with what's going on in the method.